_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -55,7 +55,6 @@ There are no assumptions.
 
 ### Accessibility Support
 
-- There are several popular browsers that do not treat images with an empty `alt` attribute (`alt=""`) as having a role of `presentation` but instead add the `img` element to the accessibility tree with a [semantic role][] of either `img` or `graphic`.
```

Comment:
Looks great!

Reply:
This has been very well supported across multiple AT/browser combinations since 2014:
https://www.powermapper.com/tests/screen-readers/labelling/img-null-alt/